e2e/libs/workload/workload.py

```diff
@@ -225,3 +229,10 @@ def count_pod_in_specifc_state_duration(count_pod_in_state_duration, pods, expec
 
 def get_pod_node(pod):
     return pod.spec.node_name
+
+def is_workload_pods_has_annotations(workload_name, annotation_key, namespace="default", label_selector=""):
+    pods = get_workload_pods(workload_name, namespace=namespace, label_selector=label_selector)
+    for pod in pods:
+        if pod.metadata.annotations and annotation_key in pod.metadata.annotations:
+            return True
```

Comment:
Looks like the statement will be satisfied if one pod has the expect annotation but the CSI plugin has many pods, should we check all pods instead of stopping after finding just one?, beside this, no other question, thank you.

Reply:
Good catch. Fixed, thank you.